clippy_lints/src/transmuting_null.rs

```diff
@@ -0,0 +1,96 @@
+use crate::consts::{constant_context, Constant};
+use crate::utils::{match_qpath, span_lint};
+use if_chain::if_chain;
+use rustc::hir::{Expr, ExprKind};
+use rustc::lint::{LateContext, LateLintPass, LintArray, LintPass};
+use rustc::{declare_tool_lint, lint_array};
+use syntax::ast::LitKind;
+
+declare_clippy_lint! {
+    pub TRANSMUTING_NULL,
```

Comment:
It's missing the lint documentation here

Reply:
That's right! :sweat_smile: I'm adding it now, thank you!

Reply:
I've added it now, please tell me if it's alright! :)